workspace_tools/tests.py

```diff
@@ -572,14 +573,14 @@
         "source_dir": join(TEST_DIR, "mbed", "can"),
         "dependencies": [MBED_LIBRARIES],
         "mcu": ["LPC1768", "LPC4088", "LPC1549", "RZ_A1H", "B96B_F446VE", "NUCLEO_F091RC",
-                "NUCLEO_F072RB", "NUCLEO_F042K6"]
+                "NUCLEO_F072RB", "NUCLEO_F042K6", "DISCO_L476VG"]
```

Comment:
Yes, we know. This L4xx family will be the last one to merged for CAN. I wanted to rebase that after F7xx family will be merged. However, thanks very much for reminding.